extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/TableCommands.scala

```diff
@@ -600,8 +600,49 @@ object TableCommands extends CommandSpecs[TableCommandSpec] {
     TableCommandSpec(cmd, Seq(tableDesc), ALTERTABLE_PROPERTIES)
   }
 
+  val FlatMapGroupsInPandas = {
+    val cmd = "org.apache.spark.sql.catalyst.plans.logical.FlatMapGroupsInPandas"
+    TableCommandSpec(cmd, Seq.empty, QUERY, queryDescs = Seq(QueryDesc("child")))
+  }
+
+  val MapInPandas = {
+    val cmd = "org.apache.spark.sql.catalyst.plans.logical.MapInPandas"
+    TableCommandSpec(cmd, Seq.empty, QUERY, queryDescs = Seq(QueryDesc("child")))
```

Comment:
Have we already visited the child in buildQuery？

Reply:
> Have we already visited the child in buildQuery？

Oh I made a mistake, pyspark's mapInPandas just uses map_partition method

Reply:
I appreciate your confirmation. Let's keep the tests if you're interested.